### PR TITLE
add route and layout to board page section

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: src/Controller/EnMarche/BoardController.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+
+/**
+ * @Route("/espace-membres-conseil")
+ * @Security("is_granted('ROLE_BOARD_MEMBER')")
+ */
+class BoardController extends Controller
```
Comment (illusionOfParadise) [inline src/Controller/EnMarche/BoardController.php]: Why not `BoardMemberController` ?

File: src/Controller/EnMarche/BoardController.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{
+    /**
+     * @Route("/", defaults={"_enable_campaign_silence"=true}, name="app_board_home")
+     * @Method("GET")
+     */
+    public function indexAction()
+    {
+        return $this->render('board/home.html.twig', [
+          'user' => $this->getUser(),
```
Comment (illusionOfParadise) [inline src/Controller/EnMarche/BoardController.php]: No need. You can use `app.user` in the template.

File: src/Controller/EnMarche/BoardController.php (hunk 1 above)
Comment (NyKo24) [reply src/Controller/EnMarche/BoardController.php]: Why not ! ;)

File: src/Controller/EnMarche/BoardController.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace AppBundle\Controller\EnMarche;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+
+/**
+ * @Route("/espace-membres-conseil")
```
Comment (illusionOfParadise) [inline src/Controller/EnMarche/BoardController.php]: What about `espace-membre-du-conseil` ?
In application the name of rest of spaces is singular.

Comment (NyKo24) [reply src/Controller/EnMarche/BoardController.php]: Marine said on clubHouse Card:
URLs
intro : /espace-membres-conseil
recherche : /espace-membres-conseil/recherche
profils sauvegardes : /espace-membres-conseil/profils-sauvegardes

File: templates/board/home.html.twig [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      <p>Nous avons créé cet espace afin que vous puissiez contacter d’autres personnes membres du conseil.</p>
+      <p>Cet outil vous sera utile si vous cherchez à monter une liste de candidature, ou bien si vous cherchez des parrainages pour vous présenter en tant que délégué(e) général(e) ou membre du bureau exécutif du conseil.  </p>
+
+      <h2>Comment ça marche ?</h2>
+      <ol>
+          <li>Commencez d’abord par remplir <a href="{{ path('app_summary_manager_index') }}">profil</a> et par ajouter votre photo.</li>
+          <li><a href="{{ path('app_board_search') }}">Rechercher des membres.</a> Vous pouvez filtrer selon leur sexe, leur role, leur
+            code postal, département ou région. Ceci vous permet de constituer des listes
+            paritaires et diverses.</li>
+          <li>Vous pouvez contacter les membres de façon groupée ou individuelle </li>
```
Comment (illusionOfParadise) [inline templates/board/home.html.twig]: Lacks a point ^^

**Pull request merged**